### PR TITLE
Move KV docs prep for secrets docs

### PR DESCRIPTION
**Pull request by victoriabernard92**
Move KV docs for runtime APIs to the reference section. Not Content change just added TOC and adjusted titles: addd14a

image

Secrets docs will then go under config vars section

### CODE REVIEW

File: content/reference/apis/_index.md [hunk 1]
```diff
@@ -4,6 +4,7 @@ alwaysopen: false
 weight: 1
 ---
 
+- [Configuration Variables](/reference/apis/config_vars)
```
Comment (1000hz) [inline content/reference/apis/_index.md]: I feel like our urls should also remain consistent with our nomenclature of `configuration_variables`

Comment (victoriabernard92) [reply content/reference/apis/_index.md]: Done

**Comment by a-robinson** [comment]
Could we please leave redirects in from the old pages to the new ones? Both in this specific case and also as a general rule?

Removing the content from the original paths is disruptive to anyone who was linking to the old pages, which in this case includes the live Workers dashboard and the Cloudflare dashboard, but likely also includes links in old forum posts on the support site, answers on StackOverflow, and other sites out there on the Internet that we don't control.

I'd add redirects myself, but I'm not sure the right way to do it in this framework. Have any pointers on how to do so?

**Comment by a-robinson** [comment]
Sorry! I see now that this did include redirects, it was purely https://github.com/cloudflare/workers-docs/pull/565 that messed things up.

**Pull request merged**